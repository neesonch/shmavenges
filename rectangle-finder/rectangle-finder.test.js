const rectangleFinder = require(`./rectangle-finder.js`);
const testRectangles = require(`./test-rectangles.js`);


test('Rectangle top left', ()=>{
	expect(rectangleFinder(testRectangles.rectangleTopLeft)).toMatchSnapshot();
})

test('Rectangle top right', ()=>{
	expect(rectangleFinder(testRectangles.rectangleTopRight)).toMatchSnapshot();
})

test('Rectangle bottom left', ()=>{
	expect(rectangleFinder(testRectangles.rectangleBottomLeft)).toMatchSnapshot();
})

test('Rectangle bottom right', ()=>{
	expect(rectangleFinder(testRectangles.rectangleBottomRight)).toMatchSnapshot();
})

test('Rectangle central', ()=>{
	expect(rectangleFinder(testRectangles.rectangleCentral)).toMatchSnapshot();
})