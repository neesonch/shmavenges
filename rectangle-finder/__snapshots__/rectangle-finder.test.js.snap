// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rectangle top left 1`] = `
Object {
  "height": 3,
  "start": Object {
    "col": 0,
    "row": 0,
  },
  "width": 4,
}
`;

exports[`Rectangle top right 1`] = `
Object {
  "height": 3,
  "start": Object {
    "col": 6,
    "row": 0,
  },
  "width": 4,
}
`;
